### ISSUE TITLE
Create page from an internal link

### ISSUE THREAD

**Issue by mpenguin** [Zettlr/zettlr-docs#165]
Would it be possible for Zettlr to be able to create pages from internal links?

I got this idea from Bryan Jenks (https://www.youtube.com/watch?v=zIh1S7ra3aI). Basically, he uses tags as umbrella categories, and [[links]] for the ability to associate ideas/topics with pages that can then be created as "hubs" for those links.

For example, if I do a lot of research into AI, I might make a lot of notes talking about [[ChatGPT]] or [[OpenAI]]. If I do that in a regular note, the page doesn't yet exist. But if I ctrl-click on it, it brings me to the search window. In Obsidian (and Roam, from what I understand), it will create a new page with that name. Then that new page can be used as a hub or page to collect all the information for that topic. The links point to the page and can be followed from any of the notes that [[link]] to it, and the page serves as the hub for the topic, linking to other notes, other websites, etc. 

I think this is a really interesting and powerful way to work, especially when combined with the Map Viewer in Obsidian (or in Zettlr). It allows you to see how many pages are linking to a certain topic; at some point, that topic page will need to be created (because with so many links, it must be important to your work/research/studies/etc.

So basically, when I ctrl-click on an [[link]] that doesn't exist, I want Zettlr to create a page with that name instead of bringing up the search. Would that be possible?

Thanks very much for your time and any thoughts and feedback.

**Comment by nathanlesage** [comment]
This is the setting you are looking for:

![image](https://user-images.githubusercontent.com/17251683/227262655-768fc698-0ee9-4b07-97ac-4401782d9dbc.png)
